cross_list_courses/templates/add_new.html

```diff
@@ -138,7 +138,7 @@ <h3>
             "bInfo" : false,
             "aoColumnDefs": [ { "orderable": false, "targets": [1,3] } ],
             "oLanguage": {
-              "sSearch": "Filter the list below"
+              "sSearch": "Filter the list"
```

Comment:
Was. this text change intended for the cross listing tool ? Or is it a search and replace by-product while fixing the self reg tool?

Reply:
Whoops -- an overly aggressive search-and-replace! :-)